conftest.py

```diff
@@ -203,6 +202,28 @@ def data_repository(directory_tree):
     return repo
 
 
+@pytest.fixture(scope='module')
+def old_repository(old_repo_name='test-renku-v0.3.0.git', tmp_path='/tmp/renku/old-repos'):
+    """Prepares a testing repo created by old version of renku."""
+    import os
+    import tarfile
+    from shutil import rmtree
+    from git import Repo
+
+    fixture_path = '{0}/tests/fixtures/{1}.tar.gz'.format(os.path.dirname(os.path.realpath(__file__)), old_repo_name)
```

Comment:
use `renku._compat:Path` for path manipulations (it's just wrapped `pathlib.Path`)

```suggestion
    fixture_path = Path(__file__).parent / 'tests' / 'fixtures' / old_repo_name
```